packages/glimmer-demos/lib/visualizer.ts

```diff
@@ -360,9 +362,10 @@ function renderContent() {
 
   let env = new TestEnvironment();
 
-  env.registerHelper('foo', function(){
-    return 'FOOOOOOO';
+  env.registerHelper('foo-bar', function(){
```

Comment:
I just changed it because I keep doing `{{#with ... as |foo|}}...{{foo}}` and `foo` ended up winning as a helper (seems like a bug??), so I named it something less common